### PR TITLE
docs sidebar nav collapse

### LINKED ISSUE #1739: Docs sidebar collapse

### PR DESCRIPTION
**Pull request by abemassry**
closes #1739 

image

image

image

### CODE REVIEW

File: docs/src/layouts/Layout.js [hunk 1]
```diff
@@ -22,14 +23,25 @@ export default class Layout extends Component {
...
+          <Link to={item.href} id={`NavLink-${index}`}>{item.label}</Link>
         </li>
       )
     });
   }
 
+  toggleVerticalNav(e) {
+    if (e.target.id === 'ToggleNavButton' || e.target.id === 'ToggleNavButtonIcon') {
+      this.setState({ verticalNav: !this.state.verticalNav });
+      console.log('toggle button clicked');
```
Comment (mparke) [inline docs/src/layouts/Layout.js]: console.log

**Comment by mparke** [comment]
mind adding a couple screenshots?

**Comment by abemassry** [comment]
@mparke screenshots done

File: docs/src/layouts/Layout.js (hunk 1 above)
Comment (abemassry) [inline docs/src/layouts/Layout.js]: good catch

**Comment by eatonphil** [comment]
screen shot 2017-06-23 at 5 03 06 pm
screen shot 2017-06-23 at 5 03 15 pm

**Comment by eatonphil** [comment]
Why are you removing python.js in this PR? Wasn't it already removed?

**Pull request merged**